core/orchestrator.go

```diff
@@ -318,6 +318,15 @@ func (n *LivepeerNode) transcodeSegmentLoop(md *SegTranscodingMetadata, segChan
 					delete(n.SegmentChans, md.ManifestID)
 				}
 				n.segmentMutex.Unlock()
+				if n.Recipient != nil {
+					sessionIDs := getPMSessionIDsByManifestID(n, md.ManifestID)
+					if len(sessionIDs) > 0 {
+						err := n.Recipient.RedeemWinningTickets(sessionIDs)
```

Comment:
I think you're right. I was having some thoughts around retries in case of an error, but any retry mechanism should probably not rely on this mapping because this ManifestID is no longer relevant anyway.

Will make this change.